Checkout/Source/Tokenisation/CheckoutAPIService.swift

```diff
@@ -59,16 +59,17 @@ final public class CheckoutAPIService: CheckoutAPIProtocol {
       riskEnvironment = .sandbox
     }
       
-    let riskConfig = RiskConfig(publicKey: publicKey, environment: riskEnvironment, framesMode: true, correlationId: logManager.correlationID)
-    let riskSDK = Risk.init(config: riskConfig)
-      
     logManager.setup(
       environment: environment,
       logger: CheckoutEventLogger(productName: Constants.Product.name),
       uiDevice: UIDevice.current,
       dateProvider: DateProvider(),
       anyCodable: AnyCodable()
     )
+    
+    let framesOptions = FramesOptions(productIdentifier: Constants.Product.name, version: Constants.Product.version, correlationId: logManager.correlationID)
```

Comment:
do you want to log the Frames version or Risk SDK version in your logs? Both of them would give information about each other since they're closely linked but still wanted to ask

Reply:
Yes, we want to log the Frames version and name on frames mode and log Risk SDK name and version on standalone mode, Pierre suggested it would help trace logs easier.